src/fragments/products.ts

```diff
@@ -28,7 +28,24 @@ export const fragmentProductImage = gql`
   }
 `;
 
+export const channelProductFragment = gql`
```

Comment:
```suggestion
export const channelListingProductFragment = gql`
```

It's much easier to use fragments named exactly like their variable names.